napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -192,29 +258,24 @@ def test_notification_error(mock_show, monkeypatch, qtbot):
     except ValueError as e:
         notif = ErrorNotification(e)
 
-    # This test creates TracebackDialog which is not added to qtbot
-    # but its parent is same as parent of NapariQtNotification.
-    # so create dummy parent allow to not leak widget.
-    # Current parent structure, where QWidget is controled by qtbot:
-    # QWidget
-    # ├── NapariQtNotification
-    # └── TracebackDialog
-    widget = QWidget()
-    qtbot.add_widget(widget)
-
-    dialog = NapariQtNotification.from_notification(notif, parent=widget)
+    dialog = NapariQtNotification.from_notification(notif)
 
     bttn = dialog.row2_widget.findChild(QPushButton)
     assert bttn.text() == 'View Traceback'
-    mock_show.assert_not_called()
+    assert count_show.show_traceback_count == 0
     bttn.click()
```

Comment:
it looks like I found a source of the problem. The click of this button is called `close_with_fade` which starts a timer. 
And just after this test, there is a segfault.